1.1/spec/09-Part-06.adoc

```diff
@@ -80,3 +61,21 @@ geo:Feature a rdfs:Class, owl:Class ;
 |`/req/core/feature-class`
 |===
 
+==== Class: geo:SpatialMeasure
+
+The class http://www.opengis.net/ont/geosparql#SpatialMeasure[`geo:SpatialMeasure`] is defined by the following:
+
+```turtle
+geo:SpatialMeasure a rdfs:Class, owl:Class ;
+    rdfs:isDefinedBy geo: ;
+    skos:prefLabel "Spatial Measure"@en ;
+    skos:definition "This class represents a measurement of some dimension 
+                    of a feature's spatial presence."@en .
+```
```

Comment:
This class has been updated recently:
:SpatialMeasure
	a owl:Class ;
  	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
	skos:definition """A measurement or estimate of one or more dimensions of a Feature's spatial presence."""@en ;
	skos:note """It can be used to convey an idea of the size of a Feature."""@en ;
	skos:prefLabel "Spatial Measure"@en ;
.